examples/lightning/checkpoint_reading.py

```diff
@@ -0,0 +1,31 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  // SPDX-License-Identifier: BSD
+
+from lightning import Trainer
+from lightning.pytorch.demos import WikiText2, LightningTransformer
+from torch.utils.data import DataLoader
+
+from s3torchconnector.lightning import S3LightningCheckpoint
+
+
+def main(region: str, checkpoint_path: str):
+    dataset = WikiText2()
+    dataloader = DataLoader(dataset, num_workers=3)
+
+    model = LightningTransformer(vocab_size=dataset.vocab_size)
+    s3_lightning_checkpoint = S3LightningCheckpoint(region)
+
+    trainer = Trainer(
+        plugins=[s3_lightning_checkpoint],
+        min_epochs=4,
+        max_epochs=5,
+        max_steps=3,
+    )
+    # Load the checkpoint in `ckpt_path` before training
```

Comment:
nit: load in CHECKPOINT_PATH

Reply:
I was thinking it makes more sense to use `ckpt_path` as that's the name of the argument as supplied by Lightning.